src/components/Molecules/Countdown/Countdown.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../../Atoms/Text/Text';
+import { Wrapper, Digits } from './Countdown.style';
+
+const Countdown = ({
+  endDate, color, endMessage, introMessage
+}) => {
+  const [countdownEnd, setCountdownEnd] = useState(false);
+  const [countdownTime, setCountdownTime] = useState({});
+
+  const countDownDate = new Date(endDate).getTime();
+  // Keep number with two digits
+  const formatNumber = number => (`0${number}`).slice(-2);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const now = new Date().getTime();
+      const distance = countDownDate - now;
+
+      const days = formatNumber(Math.floor(distance / (1000 * 60 * 60 * 24)));
+      const hours = formatNumber(Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)));
+      const minutes = formatNumber(Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60)));
+      const seconds = formatNumber(Math.floor((distance % (1000 * 60)) / 1000));
+
+      setCountdownTime({
+        days,
+        hours,
+        minutes,
+        seconds
+      });
+
+      if (distance < 0) {
+        setCountdownEnd(true);
```

Comment:
Just checking, we definitely want to hide the countdown when it's finished, rather than displaying `00:00:00:00` (and letting the parent component decide if it wants to hide it) ?

(Not sure where / how this component will be used)

Reply:
Countdown will hide when it's finished. If there is a endmessage it will display it if not it will be null.